erpnext/accounts/page/pos/pos.js

```diff
@@ -426,11 +426,11 @@ erpnext.pos.PointOfSale = erpnext.taxes_and_totals.extend({
 		});
 
 		this.serach_item.make_input();
-		this.serach_item.$input.on("keyup", function () {
-			setTimeout(function () {
+		this.serach_item.$input.on("keypress", function (event) {
+			if((me.serach_item.$input.val() != "") && (event.which == 13)){
```

Comment:
why event.which == 13?
What if user not using barcode scanner to search an item?  
After entered the search value in the search box, user needs to click on enter button every times.
Can we put or(||) condition instead of and(&&)?